Source/Tests/ExcelDna.AddIn.Tasks.IntegrationTests/ExcelDna.AddIn.Tasks.IntegrationTests.csproj

```diff
@@ -1,84 +1,16 @@
-    <FileAlignment>512</FileAlignment>
-    <RestorePackages>true</RestorePackages>
-    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
-    <TargetFrameworkProfile />
+    
+    <TargetFramework>net452</TargetFramework>
+    <UseWindowsForms>true</UseWindowsForms>
+
+    <AssemblyTitle>Excel-DNA MSBuild Tasks Integration Tests</AssemblyTitle>
+   
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <Prefer32Bit>false</Prefer32Bit>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <Prefer32Bit>false</Prefer32Bit>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="nunit.framework, Version=3.2.1.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\NUnit.3.2.1\lib\net40\nunit.framework.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="Microsoft.CSharp" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="IntegrationTestBase.cs" />
-    <Compile Include="SharedOutDirProjectsTests.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="SeparateDnaFilesCustomSuffix_x86_x64IntegrationTests.cs" />
-    <Compile Include="SeparateDnaFilesDefaultSuffixIntegrationTests.cs" />
-    <Compile Include="MsBuildParametersOverrideTests.cs" />
-    <Compile Include="PreAndPostBuildEventsIntegrationTests.cs" />
-    <Compile Include="ProjectWithNameEndingIn64IntegrationTests.cs" />
-    <Compile Include="SingleDnaFileNoConfigDefaultSuffixIntegrationTests.cs" />
-    <Compile Include="SingleDnaFileDefaultSuffixSubFolderTests.cs" />
-    <Compile Include="Utils\SilentProcessRunner.cs" />
-    <Compile Include="SingleDnaFileCustomSuffix_x64IntegrationTests.cs" />
-    <Compile Include="SingleDnaFileCustomSuffix_x86IntegrationTests.cs" />
-    <Compile Include="SingleDnaFileDefaultSuffixIntegrationTests.cs" />
-    <Compile Include="SingleDnaFileAppConfigLinkedIntegrationTests.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
-  </ItemGroup>
+
   <ItemGroup>
-    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+    <PackageReference Include="ExcelDna.Interop" Version="14.0.1" />
+    <PackageReference Include="NUnit" Version="*" />
```

Comment:
Similar to the above, I believe this `NUnit` reference should be deleted.

Reply:
Also used - keeping for now.